source/documentors/concepts/content_types.rst

```diff
@@ -1,49 +1,342 @@
+- **Be consistent** in structure, language, terminology, and tone.
+
+Criteria of a Reference Document:
+=================================
+
+.. list-table::
+   :widths: 15 40 40
+   :header-rows: 1
+
+   * - 
+     - **Criteria**
+     - 
 
-Reference topics provide details about a function or feature of the Open edX platform. For example there are many details about course subsections such as the different publication states, grading configuration, and visibility that are not included in the how-to topic *Create a Subsection* but are fully described in the reference topic *Course Subsections*.  These two topics are linked in See Also sections. 
+   * - 
+     - **Objective: Information Oriented**  
+       - Serve to Work
+     - **Technical guidelines: Theoretical Knowledge**
+
+   * - **Perfect**
+     - - The document describes the topic clearly, accurately, and comprehensively.
+       - The document is structured according to the structure of the product itself.
+     - - The document style is consistent, neutral, and objective.
+       - The document descriptions are comprehensive.
+
+   * - **Very accurate**
+     - Document shares features of Perfect and Acceptable.
+     - 
+
+   * - **Acceptable**
+     - - The document describes the content, but one (1) of the descriptions is unclear.
+       - The document follows the product's structure but omits one or two (1-2) essential points.
+     - - The document style can be more consistent, neutral, and objective.
+       - The document has one or two elements that need to be clarified.
+
+   * - **Needs corrections**
+     - Document shares features of Acceptable and Not acceptable.
+     - 
+
+   * - **Not acceptable**
+     - - Two (2) or more descriptions are not clear or accurate.
+       - The document focuses on describing a task or concept; a reference doesn't have a specific objective. It only exists to describe.
+     - - The document needs to describe the product consistently or objectively.
+       - The document descriptions could be more precise.
 
-Reference document names should use nouns and clear descriptions of the subject matter, as these docs serve as lookup resources (e.g. ``publication_states.rst``, ``grading_configuration.rst``).
 
 .. _Concept:
 
-****************
-**Concept** 
-****************
+Concept
+*******
+
+Concept documentation **clarifies and illuminates** a particular topic. It is **understanding-oriented**, and it could be considered a **conceptual guide**. Concept topics provide best practices or other guidelines for using the Open edX platform. 
+
+A Good Concept Must Have:
+=========================
+
+#. A **clear title** about the topic.
+
+  E.g., *“About XBlocks and Their Uses in Course Creation” or “What are Learning Taxonomies?”*
+
+#. An explanation that **answers a why question**. It could be descriptive, historical, or even propose different alternatives to explain the bigger picture and give context.
```

Comment:
Numbering is off

<img width="164" alt="image" src="https://github.com/user-attachments/assets/6ade1758-eee2-49dc-9827-5d4bdf17b4a2">